velox/exec/SetAccumulator.h

```diff
@@ -106,10 +153,50 @@ struct SetAccumulator {
                                  : uniqueValues.size();
   }
 
+  void serializeNullIndex(char* buffer) {
+    auto nullIndexValue =
+        nullIndex.has_value() ? nullIndex.value() : kNoNullIndex;
+    memcpy(buffer, &nullIndexValue, kVectorSizeT);
+  }
+
+  /// Extracts in result[index] a serialized VARBINARY for the Set Values.
+  /// This is used for the spill of this accumulator.
+  void serialize(const VectorPtr& result, vector_size_t index) {
+    // The serialized value is the nullOffset (kNoNullIndex if no null is
```

Comment:
This comment repeats a comment above. Let's avoid duplication as it will be hard to maintain.